src/byzerllm/saas/sparkdesk/__init__.py

```diff
@@ -60,28 +61,29 @@ def create_url(self):
         # 此处打印出建立连接时候的url,参考本demo的时候可取消上方打印的注释，比对相同参数时生成的url与自己代码生成的url是否一致
         return url
 
-class SparkDeskAPI:
+class CustomSaasAPI:
+
+    def __init__(self, infer_params: Dict[str, str]) -> None:
+        self.appid: str = infer_params.get("saas.appid","")
```

Comment:
The parameters in this three lines are required not optional. If the user miss this parameters, we should throw exception.

Reply:
OK, I will add the exception handling